docker/service/docker-compose.yaml

```diff
@@ -0,0 +1,9 @@
+version: "3.7"
+
+#TODO use variables instead of ./app dir for volumes perhaps save it in
```

Comment:
@212223 is this file needed?

Reply:
For the purpose of docker-compose commands working - not, it is not needed.

It may be deleted. If so then the only thing required is to remove `docker-compose.yaml` from the variable `COMPOSE_FILE` in [.env](./docker/service/.env) and everything works well (tested).

However it is worth to consider having it for the people that don't know docker-compose files may be merged into the final one that is used by `docker-compose` commands.

Please have a look at [docker-compose.yaml](./docker/service/docker-compose.yaml) with updated content having a brief explanation of files and differences between them.

An additional benefit of having it is that a user of this code have always a place where they may put their own rough code. A good example may be a presentation where a lecturer needs to add temporarily some bits of code to present something without messing the whole project. 
In such case there is no need to think where that code should be placed because the `docker-compose.yaml` file is something that usually comes to one's mind when they work with the docker-compose.

Would you like me to leave that docker-compose.yaml file or remove it?